src/slic3r/GUI/ConfigManipulation.cpp

```diff
@@ -345,6 +345,8 @@ void ConfigManipulation::toggle_print_fff_options(DynamicPrintConfig* config)
     toggle_field("min_feature_size", have_arachne);
     toggle_field("min_bead_width", have_arachne);
     toggle_field("thin_walls", !have_arachne);
+
+    toggle_field("min_width_top_surface", have_perimeters && config->opt_bool("only_one_perimeter_top"));
```

Comment:
Let's add `only_one_perimeter_first_layer` and `only_one_perimeter_top` to `for (auto el : { "extra_perimeters","extra_perimeters_on_overhangs", "thin_walls", "overhangs",` loop.